File: jdroid-android-core/src/debug/java/com/jdroid/android/debug/DebugSettingsActivity.java
package com.jdroid.android.debug;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class DebugSettingsActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return DebugSettingsFragment.class;
	}
}

File: jdroid-android-core/src/debug/java/com/jdroid/android/debug/DebugSettingsActivity.kt
package com.jdroid.android.debug

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class DebugSettingsActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return DebugSettingsFragment::class.java
    }
}
